src/app/api/api.ts

```diff
@@ -10,7 +10,7 @@ import { HttpClient, HttpHeaders } from '@angular/common/http';
 import { HomeService } from '../home/home.service';
 import { NetworkService } from '../network.service';
 import { ToastService } from '../toast.service';
-import { AppConfigs } from '../core-module/constants/app.config';
+import { AppConfigs } from '../core-module/constants/api.config'
```

Comment:
File name should be app.config, not api.config. make changes in all places.

Reply:
changes in all places